meal.go

```diff
@@ -175,3 +178,144 @@ func getMealDataFromID(client HttpClient, id int) (week []MealData, err error) {
 
 	return
 }
+
+var regexDateFromTitle *regexp.Regexp = regexp.MustCompile("(\\d{1,2})\\D{1,2}(\\d{1,2})\\D{1,2}(\\d{1,2})\\D{1,2}(\\d{1,2})\\D?")
+
+// 현재 시간을 기준으로 해당 주의 시간표를 추출한다.
+func getMealDataWithTime(client HttpClient, now time.Time) (week []MealData, err error) {
+	defer WhereInError(&err, "현재 주간 학식")
+
+	weekday := now.Weekday()
+	if weekday < time.Monday || weekday > time.Friday {
+		err = NotFoundError.CreateError(errors.New("주말에는 학식이 제공하지 않습니다."))
```

Comment:
Codacy found an issue: [error strings should not be capitalized or end with punctuation or a newline](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091160)